src/naming-strategy/DefaultNamingStrategy.ts

```diff
@@ -34,6 +34,10 @@ export class DefaultNamingStrategy implements NamingStrategyInterface {
         return customName ? customName : propertyName;
     }
 
+    columnAliasName(aliasName: string, columnName: string): string {
```

Comment:
Function name is little bit misleading and my first thought was it is about alias naming. Maybe something like `columnNameWithAlias`. Just opening discussion about it.